belgif-rest-problem-it/belgif-rest-problem-it-common/src/main/java/io/github/belgif/rest/problem/AbstractRestProblemIT.java

```diff
@@ -338,4 +339,39 @@ void constraintViolationBodyInheritance() {
                 .body("issues[1].detail", equalTo("must not be blank"));
     }
 
+    @Test
+    void i18n() {
+        getSpec().when()
+                .header("Accept-Language", "nl-BE")
```

Comment:
Test added. Note that we just take the one with the highest weight though, and if that one is not supported, it will fallback to English. So with Accept-Language = "fr-BE;q=0.5, es;q=0.7" the resulting language will be English, not French.